Formula/mr2.rb

```diff
@@ -0,0 +1,21 @@
+class Mr2 < Formula
+  desc "Expose local server to external network"
+  homepage "https://github.com/txthinking/mr2"
+  url "https://github.com/txthinking/mr2/archive/refs/tags/v20210401.tar.gz"
+  sha256 "3cf2874a5945e79fd9ca270181de1a9d6a662434455c58e2e20e5dbfdebd64c7"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w"
+    cd "cli/mr2" do
+      system "go", "build", *std_go_args(ldflags: ldflags)
+    end
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cli/mr2"
```
Is there a reason to prefer splitting this up over several lines?